src/components/ec-panel/ec-panel.spec.js

```diff
@@ -0,0 +1,64 @@
+import { mount } from '@vue/test-utils';
+import EcPanel from './ec-panel.vue';
+
+function mountPanel(props, mountOpts) {
+  return mount(EcPanel, {
+    propsData: { ...props },
+    ...mountOpts,
+  });
+}
+
+describe('EcPanel', () => {
+  describe(':props', () => {
```

Comment:
uhm, this is new, shall we adopt this structure in the rest of our test suite?

Reply:
Saw it here https://youtu.be/OIpfWTThrK8?t=133 was instant love.